mono_repo/lib/src/commands/github/action_info.dart

```diff
@@ -1,8 +1,20 @@
 enum ActionInfo {
-  checkout(name: 'actions/checkout', version: 'v3'),
-  cache(name: 'actions/cache', version: 'v3'),
-  setupDart(name: 'dart-lang/setup-dart', version: 'v1.3'),
-  setupFlutter(name: 'subosito/flutter-action', version: 'v2.4.0');
+  checkout(
+    name: 'actions/checkout',
+    version: 'd0651293c4a5a52e711f25b41b05b2212f385d28',
```

Comment:
Could we add a comment here referencing the release version for these shas?

Reply:
Good point! Added the version numbers in the last commit.